mlrun/api/alembic/versions/cf21882f938e_schedule_id.py

```diff
@@ -0,0 +1,29 @@
+"""schedule id
+
+Revision ID: cf21882f938e
+Revises: 11f8dd2dc9fe
+Create Date: 2020-10-07 11:21:49.223077
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'cf21882f938e'
+down_revision = '11f8dd2dc9fe'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    with op.batch_alter_table("schedules_v2") as batch_op:
+        batch_op.add_column(sa.Column('id', sa.Integer(), nullable=False))
```

Comment:
what's the magic here that let's you add a non-nullable column but still old data pass migration ?
add a comment

Reply:
It is an integer primary key. So the database knows to add sequential id numbers. Not magic - a database feature